src/app/(routes)/admin/_components/MatchManager.tsx

```diff
@@ -80,16 +72,26 @@ const MatchManager = (props: Props) => {
                   />
                 )}
               />
+              <div className="my-4 text-center text-lg">
+                Player Sessions for Match {matchIndex + 1}
+              </div>
+              {/* TODO Potentially move StatName here. Don't need to specify it for each control. */}
```

Comment:
It's fine to move statName here. I just separated PlayerSession and PlayerStats for readability/understandability when I was thinking things through.  Just need to make sure that we keep track of PlayerSession for each stat for when we insert

Reply:
Yea once we figure out the player session stuff it'll make the UI more clear.